extensions/porkbun/src/utils/types.ts

```diff
@@ -17,6 +17,7 @@ export type DomainPricing = {
           renewal?: Coupon;
           transfer?: Coupon;
         };
+    specialType?: string;
```

Comment:
I don't see any usage for this atm. Is this necessary?

```suggestion
```

Reply:
This is a new addition to the API. I planned to turn this into a "good first issue" via GitHub issues so someone who wants to try a new PR could add a Metadata Label item for it. But I can remove it.